Makefile

```diff
@@ -64,6 +64,16 @@ docker:
 	@echo ">> building docker image"
 	@docker build -t "$(DOCKER_IMAGE_NAME):$(DOCKER_IMAGE_TAG)" .
 
+k8s: promu
+	@sed -e '/linux\/amd64/{n;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;N;d;}' $(shell pwd)/.promu.yml > $(shell pwd)/.promu4travis.yml
+	@$(PROMU) --config $(shell pwd)/.promu4travis.yml crossbuild
+	@cp $(shell pwd)/.build/linux-amd64/prometheus $(shell pwd)/prometheus
+	@cp $(shell pwd)/.build/linux-amd64/promtool $(shell pwd)/promtool
+	@docker build -t prometheus:travis .
+	@echo ">> running in k8s on Travis"
+	@kubectl create configmap prometheus --from-file=$(shell pwd)/documentation/examples/
```

Comment:
just so I understand what is happening here – this imports the existing configurations as a configmap, then in the Deployment we mount this back in?